### PR TITLE
fix: updates for jaguar pilot

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-web#2232

- fix icons mapping for credit type, activities and ecosystem

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by clevinson** [comment]
The "1 credit equals" section looks like it could still use some edits to make it look a bit nicer.

@erikalogie can you input on what would be best here? It looks like for Kilo-sheep-hour credits we actually have a separate image to render the sheep logo than the sheep image that's displayed in the banner at the top of the page.

Also, I wonder if we would be better off with a smaller font size given the long sentence description of the umbrella species unit.

## KSH Credit Type Section

Screenshot 2023-12-04 at 4 42 08 PM

## USP Credit Type Section

Screenshot 2023-12-04 at 4 40 40 PM

File: web-marketplace/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx [hunk 1]
```diff
@@ -93,7 +93,8 @@ function ProjectTopSection({
   );
   const creditTypeSanity = sanityCreditTypeData?.allCreditType?.find(
     creditType =>
-      creditType.name?.toLowerCase() === creditTypeData?.creditType?.name,
+      creditType.name?.toLowerCase() ===
+      creditTypeData?.creditType?.name?.toLocaleLowerCase(),
```
Comment (blushi) [inline web-marketplace/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx:97]: why not just use `toLowerCase` here too?

Comment (flagrede) [reply web-marketplace/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx:97]: it's a mistake, thanks.

**Comment by blushi** [comment]
https://deploy-preview-2239--regen-marketplace.netlify.app/credit-classes/USP01 looks blank now, getting the following error: 
```
Uncaught TypeError: Object.defineProperty called on non-object
    at Function.defineProperty (<anonymous>)
    at requireDecUtils (index-e29176ac.js:2022:134272)
    at requireCoinUtils (index-e29176ac.js:2022:135404)
    at requireDecimal (index-e29176ac.js:2022:139288)
    at requireInt (index-e29176ac.js:2022:144424)
    at index-e29176ac.js:2022:133768
```

**Comment by flagrede** [comment]
> https://deploy-preview-2239--regen-marketplace.netlify.app/credit-classes/USP01 looks blank now, getting the following error:
> 
> ```
> Uncaught TypeError: Object.defineProperty called on non-object
>     at Function.defineProperty (<anonymous>)
>     at requireDecUtils (index-e29176ac.js:2022:134272)
>     at requireCoinUtils (index-e29176ac.js:2022:135404)
>     at requireDecimal (index-e29176ac.js:2022:139288)
>     at requireInt (index-e29176ac.js:2022:144424)
>     at index-e29176ac.js:2022:133768
> ```

I just redeploy the preview and it seems fine now.

**Review by blushi — APPROVED** [review]
LGTM, pending a decision regarding what we do about the credit type section

**Comment by clevinson** [comment]
@erikalogie Do you have a suggestion of how we could better improve this? I think the "unit" language is appropriate for what we actually write on-chain. But maybe it'd be better to have the text in this bubble be managed via Sanity? I assume that's how the images are managed as well? Or hard-coded.  I don't have a better or shorter description for the unit unfortunately. 

If we think it's confusing, then we should probably have something like "Hectare based index score" and have an option to link to the methodology if someone wants to read more details.

**Comment by blushi** [comment]
> @erikalogie Do you have a suggestion of how we could better improve this? I think the "unit" language is appropriate for what we actually write on-chain. But maybe it'd be better to have the text in this bubble be managed via Sanity? I assume that's how the images are managed as well? Or hard-coded. I don't have a better or shorter description for the unit unfortunately.
> 
> If we think it's confusing, then we should probably have something like "Hectare based index score" and have an option to link to the methodology if someone wants to read more details.

Customizing the unit language is sanity would be quick to add if we want to, but we'll still fall back to the on chain unit to support community credit classes.

**Pull request merged**